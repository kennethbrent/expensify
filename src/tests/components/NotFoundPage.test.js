import React from 'react';
import {shallow} from 'enzyme';
import toJSON from 'enzyme-to-json';
import NotFoundPage from '../../components/NotFoundPage';

test('Should render Not found page', ()=>{
    const wrapper = shallow(<NotFoundPage />)
    expect(toJSON(wrapper)).toMatchSnapshot();
})