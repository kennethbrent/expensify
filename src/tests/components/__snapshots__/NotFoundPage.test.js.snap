// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Not found page 1`] = `
<div>
  <h1>
    404!
  </h1>
</div>
`;
